packages/react/src/components/DropdownMenu/DropdownMenuItem.tsx

```diff
@@ -23,13 +25,10 @@ export const DropdownMenuItem = forwardRef<
         variant='tertiary'
         size={size}
         className='ds-dropdownmenu__item'
```

Comment:
Trenger den `ds-dropdownmenu__item`? Kan vi heller gjøre `.dropdownmenu__list > li > *` til styling?

Reply:
Eg trur den selectoren fort kan brekke om noken skal ha custom content inn i dropdownen 🤔

Reply:
Sant, kunne evt. være `:is(a,button,[role="button"])`, for man burde ikke bruke noen andre elementer enn det egentlig? ☺️

Reply:
Ja, det er nok meir skalerbart - og me slepp klassenavn :)